<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>780</width>
         <height>497</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QSlider" name="contrastR">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="contrastG">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="contrastB">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="brightnessSlider">
            <property name="minimum">
             <number>50</number>
            </property>
            <property name="maximum">
             <number>299</number>
            </property>
            <property name="value">
             <number>150</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="displayContrast">
            <property name="text">
             <string>Show</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMENU">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionShutdown"/>
   </widget>
   <widget class="QMenu" name="menuPrzetwarzanie_obraz_w">
    <property name="title">
     <string>Przetwarzanie obrazów</string>
    </property>
    <addaction name="actionSplitRgb"/>
    <addaction name="separator"/>
    <addaction name="actionProgowanie"/>
    <addaction name="actionSkala"/>
    <addaction name="actionRotate"/>
    <addaction name="separator"/>
    <addaction name="actionAveraging"/>
    <addaction name="actionGaussian"/>
    <addaction name="actionMedianAvg"/>
    <addaction name="sobelClick"/>
    <addaction name="prewittClick"/>
    <addaction name="cannyClick"/>
    <addaction name="actionDylatacja"/>
    <addaction name="actionErozja"/>
    <addaction name="actionOtwarcie"/>
    <addaction name="actionZamkniecie"/>
    <addaction name="actionGradientMorf"/>
   </widget>
   <addaction name="menuMENU"/>
   <addaction name="menuPrzetwarzanie_obraz_w"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionShutdown"/>
   <addaction name="separator"/>
   <addaction name="actionSplitRgb"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/folder-open.png</normaloff>:/icons/folder-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Otwórz</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Zapisz</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionShutdown">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/x-octagon.png</normaloff>:/icons/x-octagon.png</iconset>
   </property>
   <property name="text">
    <string>Zakończ</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionSplitRgb">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/wrench.png</normaloff>:/icons/wrench.png</iconset>
   </property>
   <property name="text">
    <string>Rozdziel na RGB</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionAveraging">
   <property name="text">
    <string>Uśrednianie</string>
   </property>
  </action>
  <action name="actionGaussian">
   <property name="text">
    <string>Rozmycie Gausowskie</string>
   </property>
  </action>
  <action name="actionProgowanie">
   <property name="text">
    <string>Progowanie obrazu</string>
   </property>
  </action>
  <action name="actionSkala">
   <property name="text">
    <string>Skala</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="text">
    <string>Obrót</string>
   </property>
  </action>
  <action name="actionMedianAvg">
   <property name="text">
    <string>Uśrednianie medianowe</string>
   </property>
  </action>
  <action name="sobelClick">
   <property name="text">
    <string>Detekcja krawędzi m. Sobela</string>
   </property>
  </action>
  <action name="prewittClick">
   <property name="text">
    <string>Detekcja krawędzi m. Prewitta</string>
   </property>
  </action>
  <action name="cannyClick">
   <property name="text">
    <string>Detekcja krawędzi m. Canny'ego</string>
   </property>
  </action>
  <action name="actionDylatacja">
   <property name="text">
    <string>Dylatacja</string>
   </property>
  </action>
  <action name="actionErozja">
   <property name="text">
    <string>Erozja</string>
   </property>
  </action>
  <action name="actionOtwarcie">
   <property name="text">
    <string>Otwarcie</string>
   </property>
  </action>
  <action name="actionZamkniecie">
   <property name="text">
    <string>Zamknięcie</string>
   </property>
  </action>
  <action name="actionGradientMorf">
   <property name="text">
    <string>Gradient Morfologiczny</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
